shell/common/node_includes.h

```diff
@@ -15,7 +15,6 @@
 #undef require_string  // This is defined in macOS SDK in AssertMacros.h.
 
 #include "electron/push_and_undef_node_defines.h"
```

Comment:
This change works but is a little weird; grouping it with the node headers below implies they're all the same rank but this one _must_ be #included first so it should be on its own. The way it is in this PR works only because `electron/push_and_undef_node_defines.h` comes before `env-inl.h` alphabetically